src/pages/inquiry/list/index.tsx

```diff
@@ -29,7 +26,7 @@ const InquiryList: React.FC = () => {
 	// Fetch events information
 	useEffect(() => {
 		dispatch(getContactsUs());
-	}, []);
+	}, [dispatch, contactsUs]);
```

Comment:
The change should be like this. You need to pass the `getContactsUs()` as the second item in the array. Not the `contactsUs` array.
```
useEffect(() => {
   dispatch(getContactsUs());
}, [dispatch, getContactsUs]);
```